erts/preloaded/src/erlang.erl

```diff
@@ -962,7 +963,8 @@ float_to_list(_Float) ->
       Options :: [Option],
       Option  :: {decimals, Decimals :: 0..253} |
                  {scientific, Decimals :: 0..249} |
-                 compact.
+                 compact |
+                 pretty_print.
```

Comment:
The documentation describes the option `short` but here it seems like it is called `pretty_print`.

Reply:
that is outdated i think, i have it as short on my branch rn ?

Reply:
I clicked on the commit 4e0c6525a670f9011f08999400df16b2a1b31abe (add option short to float_to_*/2 bifs) but this small change was in 886157407c59d43f06c5b5aac1f41863accc83d1 (update preloaded) and in 027a545fa60063303b4e0fbc4f03563762a35266 (test short option in num_bif_SUITE ) so I missed that. Can you squash the commits into one so everything is in the same commit? It seems like this change was the only one that I missed.